frontend/app/src/pages/balances/exchange/index.vue

```diff
@@ -172,7 +227,11 @@ const { t, tc } = useI18n();
 </template>
 
 <style scoped lang="scss">
-:deep(.v-tabs-bar) {
+/* stylelint-disable selector-class-pattern */
+
+:deep(.exchange-balances__tabs) {
```

Comment:
This is our own class right? why use deep? is it in another component?

Reply:
I'm targeting the tab component. Anyway it's moved now